src/app/lib/layoutGrid.js

```diff
@@ -13,36 +13,43 @@ import {
 export const layoutGridWrapper = css`
   display: grid;
   @media (max-width: ${group3ScreenWidthMin}) {
-    grid-gap: ${GEL_SPACING}; /* Between grid items */
-    padding: 0 ${GEL_SPACING}; /* On grid wrapper */
+    grid-gap: 0 ${GEL_SPACING}; /* Between grid items */
   }
   @media (min-width: ${group3ScreenWidthMin}) {
-    grid-gap: ${GEL_SPACING_DBL}; /* Between grid items */
-    padding: 0 ${GEL_SPACING_DBL}; /* On grid wrapper */
+    grid-gap: 0 ${GEL_SPACING_DBL}; /* Between grid items */
   }
   @media (max-width: ${group3ScreenWidthMax}) {
-    grid-template-columns: repeat(6, 1fr);
+    grid-template-columns: repeat(6, [col] 1fr);
   }
   @media (min-width: ${group4ScreenWidthMin}) and (max-width: ${group4ScreenWidthMax}) {
-    grid-template-columns: repeat(8, 1fr);
+    /* 
+      Math: (1008 - (7*16))/8 = 112
+      Explainer: (width - 7 gaps of 16) / 8 columns = single column width
```

Comment:
I'm unsure on how this would work as we are using fractional units to handle the flexible columns either side of the 10 "article columns" we have in the central 1280px. If the 10 "article columns" are fractional units and the flexible columns are fractional units then we will have 12 fractional units and then lose the 1280px structure which we make 10 columns of. 

I've created a playground codepen for us to trial different implementations - https://codepen.io/phil-lee/pen/zmjaMO

Reply:
Discussed offline, attempted to use `grid-template-columns: auto repeat(10, 1fr) 1fr auto` however this renders with the columns taking the viewport and splitting it into 10 equal columns with the remainder being renderer in the first auto and the 2nd auto being treated as nothing. This therefore does not maintain the 10 columns of 1280px so isn't a viable option. 

Outcome being that we are happy to continue with the explicit column width setting within the repeat